docs/api/createSlice.md

```diff
@@ -82,6 +82,8 @@ to force the TS compiler to accept the computed property.)
 Each function defined in the `reducers` argument will have a corresponding action creator generated using [`createAction`](./createAction.md)
```

Comment:
Looks like the formatting check failed.  Please run the changes through Prettier.